docs/LEARNING.md

```diff
@@ -1,15 +1,14 @@
-## Learning Perl 6
+## Learning Raku (formerly Perl 6)
```

Comment:
In https://github.com/exercism/perl6/pull/353 I've mentioned in the first sentence of ABOUT.md that the language's former name is Perl 6. How prominent should we make this? IMO I don't think it needs to be a header.